sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/PathAccessControlEntry.java

```diff
@@ -0,0 +1,307 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.file.datalake.models;
+
+import com.azure.storage.common.implementation.StorageImplUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+import java.util.Objects;
+
+/**
+ * POSIX access control rights on files and directories.
+ * <p>
+ * The value is a comma-separated list of access control entries, each access control entry (ACE) consists of four
+ * elements in the format "[scope:][type]:[id]:[permissions]":
+ * <ul>
+ *   <li>Scope</li>
+ *   <li>Type</li>
+ *   <li>User or Group Identifier</li>
+ *   <li>Permissions</li>
+ * </ul>
+ * <p>
+ * The scope must be "default" to indicate the ACE belongs to the default Access Control List (ACL) for a directory;
+ * otherwise scope is implicit and the ACE belongs to the access ACL.
+ * <p>
+ * There are four ACE types:
+ * <ul>
+ *   <li>"user": grants rights to the owner or a named user</li>
+ *   <li>"group" grants rights to the owning group or a named group</li>
+ *   <li>"mask" restricts rights granted to named users and the members of groups</li>
+ *   <li>"other" grants rights to all users not found in any of the other entries</li>
+ * </ul>
+ * <p>
+ * The user or group identifier is omitted for entries of type "mask" and "other".
```

Comment:
Can we mention here (and on constructors/setters) that the id is a AAD ObjectId?

Reply:
Added it here. Looks like it is already specified in the getter/setter